bin/amphtml-update.py

```diff
@@ -953,7 +947,7 @@ def Phpize(data, indent=0):
 
 	# Clean up formatting.
 	# TODO: Just use PHPCBF for this.
-	php_exported = re.sub( r'^ +', lambda match: ( len(match.group(0))/2 ) * '\t', php_exported, flags=re.MULTILINE )
+	php_exported = re.sub( r'^ +', lambda match: ( round(len(match.group(0))/2) ) * '\t', php_exported, flags=re.MULTILINE )
```

Comment:
This change here is causing an error for me:

```
Traceback (most recent call last):
    php_exported = re.sub( r'^ +', lambda match: ( round(len(match.group(0))/2) ) * '\t', php_exported, flags=re.MULTILINE )
TypeError: can't multiply sequence by non-int of type 'float'
```

This is fixed for me by doing:

```suggestion
	php_exported = re.sub( r'^ +', lambda match: int( round(len(match.group(0))/2) ) * '\t', php_exported, flags=re.MULTILINE )
```